### PR TITLE
[Merged by Bors] - feat(Algebra/Category/ModuleCat): the category of presheaves of modules has limits

### PR DESCRIPTION
**Pull request by joelriou**
---

- [x] depends on: #11669

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Category/ModuleCat/Presheaf/Limits.lean [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+variable {C : Type u₁} [Category.{v₁} C] {R : Cᵒᵖ ⥤ RingCat.{u}}
+  {J : Type u₂} [Category.{v₂} J]
+  (F : J ⥤ PresheafOfModules.{v} R)
+
+section Limits
+
+variable [∀ X, Small.{v} ((F ⋙ evaluation R X) ⋙ forget _).sections]
+
+/-- A cone in the category `PresheafOfModules R` is limit if it is so after the application
+of the functors `evalurtion R X` for all `X`. -/
```
Comment (erdOne) [inline Mathlib/Algebra/Category/ModuleCat/Presheaf/Limits.lean:36]:
```suggestion
of the functors `evaluation R X` for all `X`. -/
```

**Review by erdOne — COMMENTED** [review]
Otherwise LGTM

File: Mathlib/Algebra/Category/ModuleCat/Presheaf/Limits.lean (hunk 1 above)
Comment (joelriou) [reply Mathlib/Algebra/Category/ModuleCat/Presheaf/Limits.lean:36]: Thanks for the review!

**Comment by erdOne** [comment]
maintainer merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**